trade-finance-manager-ui/templates/utilisation-reports/_macros/report-reconciliation-table-manual-reconciliation.njk

```diff
@@ -0,0 +1,107 @@
+{% import "./report-reconciliation-status.njk" as reconciliationStatus %}
+{% import "../../_macros/table-cell-checkbox.njk" as tableCellCheckbox %}
+
+{% macro render(params) %}
+  {% set user = params.user %}
+  {% set summaryItems = params.summaryItems %}
+  {% set submissionMonth = params.submissionMonth %}
+  {% set csrfToken = params.csrfToken %}
+
+  <form method="POST" data-cy="utilisation-reports-form--{{ submissionMonth }}">
+    <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
+    <table class="govuk-table" data-module="moj-sortable-table" data-cy="utilisation-report-reconciliation-table">
+      <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th scope="col" class="govuk-table__header" aria-sort="ascending">Bank</th>
+          <th scope="col" class="govuk-table__header" aria-sort="none">Status</th>
+          <th scope="col" class="govuk-table__header" aria-sort="none">Date report received</th>
+          <th scope="col" class="govuk-table__header" aria-sort="none">Total fees reported</th>
+          <th scope="col" class="govuk-table__header" aria-sort="none">Reported fees left to reconcile</th>
+          <th scope="col" class="govuk-table__header">Download report as CSV</th>
+          <th scope="col" class="govuk-table__header">Select</th>
+        </tr>
+      </thead>
+
+      <tbody class="govuk-table__body">
+      {% for summaryItem in summaryItems %}
+        {% set checkboxId = ['set-status--reportId-', summaryItem.reportId, '-currentStatus-', summaryItem.status] | join %}
+        <tr
+          class="govuk-table__row"
+          data-cy="utilisation-report-reconciliation-table-row-bank-{{ summaryItem.bank.id }}-submission-month-{{ submissionMonth }}"
+        >
+          <th
+            scope="row"
+            class="govuk-table__header govuk-!-font-weight-regular"
+            data-sort-value="{{ summaryItem.bank.name }}"
+          >
+            <p class="govuk-body govuk-!-margin-0">
+                {{ summaryItem.bank.name }}
+            </p>
+          </th>
+          <td class="govuk-table__cell" data-sort-value="{{ summaryItem.displayStatus }}">
+            {{ reconciliationStatus.render({
+              status: summaryItem.status,
+              displayStatus: summaryItem.displayStatus
+            }) }}
+            </td>
+            <td class="govuk-table__cell" data-sort-value="{{ summaryItem.dateUploaded }}">
+              {{ summaryItem.formattedDateUploaded }}
+            </td>
+            <td class="govuk-table__cell" data-sort-value="{{ summaryItem.totalFeesReported }}">
+              {{ summaryItem.totalFeesReported }}
+            </td>
+            <td class="govuk-table__cell" data-sort-value="{{ summaryItem.reportedFeesLeftToReconcile }}">
+              {% if summaryItem.status === 'RECONCILIATION_COMPLETED' %}
+                0
+              {% else %}
+                {{ summaryItem.totalFeesReported }}
+              {% endif %}
```

Comment:
This might be an actual terms change but it felt wrong for a report to be allowed to be complete and have fees left to reconcile? That said, I'm not sure what we are going to do for old reports when we turn the feature flag on, @ChristianMcc do you know if we have a ticket to cover not allowing users to go to premium reports page etc for reports that were set to reconciled manually from before we had the reconciliation journey?

Reply:
(Also I haven't written tests for this yet but will do so on Monday)